cmd/cloud-endpoints-controller/main.go

```diff
@@ -481,3 +519,16 @@ func getIngBackends(ing *v1beta1.Ingress) ([]string, error) {
 func makeJWTAudience(projectNum, backendID string) string {
 	return fmt.Sprintf("/projects/%s/global/backendServices/%s", projectNum, backendID)
 }
+
+func toSha1(data string) string {
+	h := sha1.New()
+	h.Write([]byte(data))
+	hashed := hex.EncodeToString(h.Sum(nil))
```

Comment:
No need to create a new variable `hashed`m just return the output of the function call.